src/cli.rs

```diff
@@ -75,6 +75,10 @@ pub struct Opts {
     #[arg(long, env = "PROBE_RUN_PROBE")]
     pub probe: Option<String>,
 
+    /// Whether to reset the target when attaching and detaching. Implies `--no-flash`.
+    #[arg(long = "no-reset", default_value = "true", action = ArgAction::SetFalse)]
+    pub reset: bool,
```

Comment:
I prefer to keep the field name and the name of the cli option the same. Otherwise it could get confusing.

```suggestion
    #[arg(long)]
    pub no_reset: bool,
```

Please maintain the alphabetical order when you change the name.